diff --color -urN udisks-2.9.1.orig/src/udiskslinuxprovider.c udisks-2.9.1/src/udiskslinuxprovider.c
--- udisks-2.9.1.orig/src/udiskslinuxprovider.c	2020-08-12 16:05:39.000000000 +0200
+++ udisks-2.9.1/src/udiskslinuxprovider.c	2020-11-13 01:06:57.098257205 +0100
@@ -720,8 +720,8 @@
   g_list_free_full (udisks_devices, g_object_unref);
   udisks_info ("Initialization complete");
 
-  /* schedule housekeeping for every 10 minutes */
-  provider->housekeeping_timeout = g_timeout_add_seconds (10*60,
+  /* schedule housekeeping for every 60 minutes */
+  provider->housekeeping_timeout = g_timeout_add_seconds (60*60,
                                                           on_housekeeping_timeout,
                                                           provider);
   /* ... and also do an initial run */
@@ -1502,7 +1502,7 @@
   G_UNLOCK (provider_lock);
 }
 
-/* called from the main thread on start-up and every 10 minutes or so */
+/* called from the main thread on start-up and every 60 minutes or so */
 static gboolean
 on_housekeeping_timeout (gpointer user_data)
 {
