--- udisks-2.11.0/src/udiskslinuxprovider.c.orig	2025-11-06 14:43:45.000000000 +0100
+++ udisks-2.11.0/src/udiskslinuxprovider.c	2026-02-21 20:37:27.252988169 +0100
@@ -794,8 +794,8 @@ udisks_linux_provider_start (UDisksProvi
   g_list_free_full (udisks_devices, g_object_unref);
   udisks_info ("Initialization complete");
 
-  /* schedule housekeeping for every 10 minutes */
-  provider->housekeeping_timeout = g_timeout_add_seconds (10*60,
+  /* schedule housekeeping for every 60 minutes */
+  provider->housekeeping_timeout = g_timeout_add_seconds (60*60,
                                                           on_housekeeping_timeout,
                                                           provider);
   /* ... and also do an initial run */
@@ -1592,7 +1592,7 @@ housekeeping_thread_func (GTask
   g_task_return_boolean (task, TRUE);
 }
 
-/* called from the main thread on start-up and every 10 minutes or so */
+/* called from the main thread on start-up and every 60 minutes or so */
 static gboolean
 on_housekeeping_timeout (gpointer user_data)
 {
